src/components/ConversationSettings/RecordingConsentSettings.vue

```diff
@@ -0,0 +1,133 @@
+<!--
+  - @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @author Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="app-settings-subsection">
+		<h4 class="app-settings-section__subtitle">
+			{{ t('spreed', 'Recording Consent') }}
+		</h4>
+
+		<label for="recording-consent_input" class="recording-consent__label">
+			{{ t('spreed', 'Require recording consent before joining call in this conversation') }}
+		</label>
+		<NcSelect v-model="recordingConsentSelected"
```

Comment:
Don't show the select when it's enforced on admin level

Reply:
Should we do it the same as for Expiration settings? https://github.com/nextcloud/spreed/pull/10491

Reply:
Sounds good.